projects/ReOxide/services/reoxided/module.nix

```
@@ -0,0 +1,73 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    mkIf
+    types
+    ;
+
+  cfg = config.services.reoxided;
+  toToml = pkgs.formats.toml { };
+  configFile = toToml.generate "reoxide.toml" {
+    ghidra-install = cfg.ghidraInstall;
+  };
```

We can put this in a `services.reoxided.settings` option and just pass that, here. No need for repetition.


```suggestion
  configFile = toToml.generate "reoxide.toml" cfg.settings;
```